apps/s_client.c

```diff
@@ -787,6 +788,7 @@ const OPTIONS s_client_options[] = {
 #endif
     {"keylogfile", OPT_KEYLOG_FILE, '>', "Write TLS secrets to file"},
     {"early_data", OPT_EARLY_DATA, '<', "File to send as early data"},
+    {"force_pha", OPT_FORCE_PHA, '-', "Force-enable post-handshake-authentication"},
```

Comment:
This option (and the s_server equivalent) needs adding to the docs

Reply:
Added to docs, s_server option removed.